ruby/rails/security/audit/avoid-tainted-shell-call.yaml

```diff
@@ -1,7 +1,7 @@
 rules:
   - id: avoid-tainted-shell-call
     metadata:
-      owasp: "A1: Injection"
+      owasp: "A01:2017 - Injection"
```

Comment:
should maybe add A03:2021 - Injection as well?

Reply:
sounds good to me :D

Reply:
done!